spec/lang/values.md

```diff
@@ -481,7 +466,7 @@ impl<M: Memory> TypedMemory for M {
             _ => panic!("this value does not have that type"),
         })
     }
-}
+}  
```

Comment:
```suggestion
}
```

Reply:
I found 1457 occurrences of `\s\n` some of those in lock files. Should I fix them or just pay attention in the future?

Reply:
`\s` finds lots of nonsense matches for me; looking for ` \n` I only find
```
spec/lang/locks.md:1
spec/lang/step.md:4
spec/lang/machine.md:2
spec/lang/intrinsic.md:1
tooling/minitest/src/ub/join.rs:4
tooling/minitest/src/ub/spawn.rs:5
tooling/miniutil/src/build/statement.rs:25
tooling/minitest/src/ub/locks.rs:55
```
Would be good to fix, but in a separate PR please. Just this PR shouldn't add more of them :)

Reply:
Ah, `\s\n` also matches `\n\n` which of course we have a ton of

Reply:
Oh, my bad.